contracts/MemeBase.sol

```diff
@@ -216,4 +230,9 @@ contract MemeBase is MemeFactory {
             }
         }
     }
+
+    function _wrap(uint256 nativeTokenAmount) internal virtual override {
```

Comment:
This implementation should be on this level, not factory

Reply:
Fine, but the default one most time will be the `_wrap()` like ETH to WETH, and not CELO.